posthog/batch_exports/service.py

```diff
@@ -687,7 +687,7 @@ def sync_batch_export(batch_export: BatchExport, created: bool):
             start_at=batch_export.start_at,
             end_at=batch_export.end_at,
             intervals=[ScheduleIntervalSpec(every=batch_export.interval_time_delta)],
-            jitter=(batch_export.interval_time_delta / 6),
+            jitter=max(dt.timedelta(seconds=60), (batch_export.interval_time_delta / 6)),
```

Comment:
This means 5 minute batch exports will start between 0 and 60 seconds after the 5 minute mark. Unfortunately, we can't set a minimum jitter, otherwise I would have set this to 60 seconds _minimum_.

Hourly and daily batch exports are unaffected as their jitters are already larger.